schema/va-spec/base/va-core-source.yaml

```diff
@@ -557,7 +557,6 @@ $defs:
         ordered: false
         items:
           anyOf:
-            - $ref: "#/$defs/StudyResult"
```

Comment:
@ga4gh/va-admin I removed this. I was getting `jsonschema.exceptions._WrappedReferencingError: Unresolvable: /ga4gh/schema/va-spec/1.0.0-ballot.2024-11.2/base/json/StudyResult` since this seems to be an abstract class.